beacon-chain/rpc/proposer_server.go

```diff
@@ -164,12 +143,37 @@ func (ps *ProposerServer) eth1Data(ctx context.Context, slot uint64) (*ethpb.Eth
 	// Look up most recent block up to timestamp
 	blockNumber, err := ps.eth1BlockFetcher.BlockNumberByTimestamp(ctx, eth1VotingPeriodStartTime)
 	if err != nil {
-		return nil, err
+		return ps.mockETH1DataVote(slot)
 	}
 
 	return ps.defaultEth1DataResponse(ctx, blockNumber)
 }
 
+func (ps *ProposerServer) mockETH1DataVote(slot uint64) (*ethpb.Eth1Data, error) {
```

Comment:
This function should issue a `WARN` level log specifying the eth1 chain was down during this time and that we are using a mock eth1 data vote instead, what do you think?

Reply:
Thinking about how do we represent this to a validator. This would just be a log warning in the beacon node and
can't be relayed to a validator.

Reply:
But our beacon node should at least have some sort of log, maybe in the powchain service that says powchain is offline and validators will be given mock data in the meantime?

Reply:
Sounds good, I will add it in